src/jwt/model.rs

```diff
@@ -16,50 +15,36 @@ pub struct Claims {
     pub iss: String,
     // subject
     pub sub: String,
-    //issued at
+    // issued at
     pub iat: i64,
     // expiry
     pub exp: i64,
     // user email
     pub email: String,
+    // user role
+    pub role: String,
 }
 
-pub type HumanClaims = Claims;
```

Comment:
Here, I don't think that the Claims should be seen as Human-printable from the model-layer, but should be Human-printable on the presentation-layer... thus I mostly move this in `ClaimsResponse`.

Reply:
I think also, thanks